### ISSUE TITLE
CoreCLR can't handle RealParse.cs code

### ISSUE THREAD

**Issue by jaredpar** [dotnet/roslyn#5870]
The CoreCLR crashes with exit code 6 when it attempts to run @gafter double parsing code on Linux.  In the short term the code has been replaced with the old `double.TryParse` and `float.TryParse` routines when compiling for Linux / Mac.  It needs to be fixed to use the actual RealParse code.

**Comment by gafter** [comment]
This is related to https://github.com/dotnet/roslyn/pull/5379 which has not yet been pulled because of this issue.

**Comment by jaredpar** [comment]
@gafter I created a new PR #5871 which combines #5379 with the work around for this bug.  I couldn't update your PR directly and wanted to get this in the lab to see if there were any other issues remaining.  Don't think there will be as it passed on my machine.

**Comment by janvorli** [comment]
@jaredpar, @gafter I've fixed some issues in exception handling that may be related last Wednesday and Thursday in CoreCLR:
https://github.com/dotnet/coreclr/pull/1694
https://github.com/dotnet/coreclr/pull/1715
If it doesn't fix the issue, would it be possible to get a simple repro for the issue?

**Comment by janvorli** [comment]
Also, could you please CC me on all Linux CoreCLR issues you hit in the future?

**Comment by agocke** [comment]
@janvorli Yup, thanks Jan, I'll try to update to a new CoreCLR build and see if that fixes the issue. Unfortunately, repros are hard to come by due to the lack of debugging information on Linux.

**Comment by agocke** [comment]
@janvorli Just FYI, this appears to be Linux-specific as I have just tried rebuilding with CoreCLR on Windows and did not hit any compilation issues.

**Comment by agocke** [comment]
@janvorli Never mind, I was testing with a fixed commit, I'll retest with the failing version.

**Comment by agocke** [comment]
@janvorli I believe I've nailed down the problem -- we were missing System.Runtime.Numerics, which appears to be necessary at runtime. I'll close this issue once #5871 is merged successfully.

**Comment by agocke** [comment]
Closed by #5871
